src/python/zquantum/qaoa/estimators.py

```diff
@@ -1,15 +1,16 @@
 import numpy as np
 from openfermion import IsingOperator
-from typing import Optional
-from overrides import overrides
+from typing import List
 
-from zquantum.core.circuit import Circuit
-from zquantum.core.interfaces.backend import QuantumBackend, QuantumSimulator
+from zquantum.core.interfaces.backend import QuantumBackend
 from zquantum.core.measurement import ExpectationValues, Measurements
-from zquantum.core.interfaces.estimator import Estimator
+from zquantum.core.wip.estimators.estimation_interface import (
+    EstimateExpectationValues,
+    EstimationTask,
+)
 
 
-class CvarEstimator(Estimator):
+class CvarEstimator(EstimateExpectationValues):
```

Comment:
Wouldn't it make more sense for this to be a function similar to `estimate_expectation_values_by_averaging`? Not that a class implementation wouldn't work, but it seems like the only difference is having the `alpha` parameter, which could be taken care of using `partial` functions.

Although I don't have very strong feelings about this, so open to hearing feedback.